javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,110 @@
+let userName = ""; // Rodrigo;
+const todoList = [];
+let lettersToCount = [];
+
+function getReply(command) {
+  if (command == " " || typeof command !== "string") {
+    alert(`command is not a string`);
+  }
+
+  if (command.includes("Hello my name is")) {
+    const arrayOfInputString = command.split(" ");
+    let savedName = arrayOfInputString[4];
+    if (savedName === userName) {
+      console.log("We met before");
+    } else {
+      userName = savedName;
+      console.log(`Nice to meet you ${savedName}`);
+    }
+  }
+
+  if (command.includes("What is my name") && userName === "") {
+    console.log("You did not tell me your name");
+  } else if (command.includes("What is my name")) {
+    console.log(`Your name is ${userName}`);
+  }
+
+  if (command.includes("Add") && command.includes("to my todo")) {
+    const myTodo = command.slice(4, -11);
+    todoList.push(myTodo);
+    console.log(`${myTodo} added to your todo`);
+  }
+
+  if (command.includes("Remove")) {
+    const removeFromTodo = command.slice(7, -13);
+    for (let i = 0; i < todoList.length; i++) {
+      if (todoList[i] === removeFromTodo) {
+        todoList.splice(i, 1);
+      }
+    }
+  }
+
+  if (command === "What is on my todo?") {
+    for (let i = 0; i < todoList.length; i++) {
+      console.log(todoList[i]);
+    }
+  }
+  if (command === "What day is it today?") {
+    const today = new Date();
+    const day = today.toLocaleString("en-US", { day: "2-digit" });
+    const month = today.toLocaleString("en-US", { month: "long" });
+    const year = today.toLocaleString("en-US", { year: "numeric" });
+    const date = `${day}. of ${month} ${year}`;
+    console.log(date);
+  }
+
+  if (command.includes("What is") && command.includes("+")) {
```

Comment:
this could also be extracted into its own function. Here you could do like a function called calculate and then pass the +, -, *, / and so on depending on the user input.